use/developer/uci-basics/adapters/firebase-notification-adapter.md

```diff
@@ -88,6 +88,30 @@ We can store these registered tokens in UCI & use UCI fetch api to get these tok
     ```
     curl --location --request GET 'http://143.110.255.220:9999/campaign/start?campaignId=1ea5346d-8d98-4cf4-a470-9c234476f3d1'
     ```
+The table below explains all the request keys while creating a conversation logic along with their description.
+
+| Key | Type | Required | Description |
+| --- | --- | --- | --- |
+| data | object | Yes | The root object of the data. |
+| data.id | null | No | The id of the data. |
```

Comment:
What should be the format of the ID?

Reply:
Can you help me out with this? The example request you shared had null in this field.

Reply:
This field is not being used and the `id` corresponds to the id of the bot in the database which is autogenerated.